velox/functions/prestosql/aggregates/tests/ArbitraryTest.cpp

```diff
@@ -367,5 +367,59 @@ TEST_F(ArbitraryTest, interval) {
   testAggregations({data}, {}, {"arbitrary(c2)"}, "SELECT null");
 }
 
+TEST_F(ArbitraryTest, longDecimal) {
+  auto data = makeRowVector({// Grouping key.
+                             makeFlatVector<int64_t>({1, 1, 2, 2, 3, 3, 4, 4}),
+                             makeNullableFlatVector<int128_t>(
+                                 {HugeInt::build(10, 100),
+                                  HugeInt::build(10, 100),
+                                  HugeInt::build(10, 200),
+                                  HugeInt::build(10, 200),
+                                  std::nullopt,
+                                  std::nullopt,
+                                  std::nullopt,
+                                  HugeInt::build(10, 400)},
+                                 DECIMAL(38, 8))});
+
+  auto expectedResult = makeRowVector({
```

Comment:
Would you please verify expected null behavior against Presto?
As per definition here `arbitrary()` returns non-null values:https://prestodb.io/docs/current/functions/aggregate.html#general-aggregate-functions